docs/source/hyperparameters.rst

```diff
@@ -112,87 +112,87 @@ Often times we train many versions of a model. You might share that model or com
 at which point it is very useful to know how that model was trained (i.e.: what learning rate, neural network, etc...).
 
 Lightning has a few ways of saving that information for you in checkpoints and yaml files. The goal here is to
-improve readability and reproducibility
+improve readability and reproducibility.
 
-1. The first way is to ask lightning to save the values of anything in the __init__ for you to the checkpoint. This also
-makes those values available via `self.hparams`.
+1.  The first way is to ask lightning to save the values of anything in the __init__ for you to the checkpoint. This also
+    makes those values available via `self.hparams`.
 
-.. code-block:: python
+    .. code-block:: python
```

Comment:
could it be rather a test code? it shall pass as it would just test the syntax and func references...